test/integration/api-schema/basic.js

```diff
@@ -24,50 +24,74 @@ const next = afterAll(function (err, validators) {
   test('metadata schema failure', function (t) {
     t.strictEqual(validateMetadata({}), false)
     validateFieldMessages(t, validateMetadata.errors, [
-      { field: 'data.service', message: 'is required' }
+      { message: 'should have required property \'service\'' }
     ])
     t.end()
   })
 
   test('transaction schema failure', function (t) {
     t.strictEqual(validateTransaction({}), false)
     validateFieldMessages(t, validateTransaction.errors, [
-      { field: 'data.duration', message: 'is required' },
-      { field: 'data.type', message: 'is required' },
-      { field: 'data.id', message: 'is required' },
-      { field: 'data.trace_id', message: 'is required' },
-      { field: 'data.span_count', message: 'is required' }
+      { message: 'should have required property \'duration\'' },
+      { message: 'should have required property \'id\'' },
+      { message: 'should have required property \'span_count\'' },
+      { message: 'should have required property \'trace_id\'' },
+      { message: 'should have required property \'type\'' }
     ])
```

Comment:
I presume the format of these objects have change to accommodate the behavior of the new validator.  There's a 1:1 for each `data.` properties from the old format into the new format. 👍